creme/drift/page_hinkley.py

```diff
@@ -70,38 +72,36 @@ def reset(self):
         self.x_mean = 0.0
         self.sum = 0.0
 
-    def add_element(self, x):
+    def update(self, x):
         """ Add a new element to the statistics
-        
+
         Parameters
         ----------
         x: numeric value
             The observed value, from which we want to detect the
             concept change.
-        
+
         Notes
         -----
-        After calling this method, to verify if change was detected, one 
-        should call the super method detected_change, which returns True 
+        After calling this method, to verify if change was detected, one
+        should call the super method detected_change, which returns True
```

Comment:
We could change that to reflect the new access interface to the drift detection methods.

Reply:
My bad, I removed this text and updated the documentation of the drift detectors in 128eb08c6e89b60e039a8029b272f6dd35a984cb